cmd/admin-config-get.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+
+	"github.com/minio/cli"
+	"github.com/minio/minio/pkg/probe"
+)
+
+var (
+	adminConfigGetFlags = []cli.Flag{}
+)
+
+var adminConfigGetCmd = cli.Command{
+	Name:   "get",
+	Usage:  "Get config of a Minio server/cluster.",
+	Before: setGlobalsFromContext,
+	Action: mainAdminConfigGet,
+	Flags:  append(adminConfigGetFlags, globalFlags...),
+	CustomHelpTemplate: `NAME:
+   {{.HelpName}} - {{.Usage}}
+
+USAGE:
+   {{.HelpName}} ALIAS/
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+    1. Get server configuration of a Minio server/cluster.
+       $ {{.HelpName}} play/
+
+`,
+}
+
+// configGetMessage container to hold locks information.
+type configGetMessage struct {
+	Status string `json:"status"`
+	Config []byte `json:"config"`
+}
+
+// String colorized service status message.
+func (u configGetMessage) String() string {
+	return string(u.Config)
+}
+
+// JSON jsonified service status Message message.
+func (u configGetMessage) JSON() string {
+	u.Status = "success"
+	statusJSONBytes, e := json.Marshal(u)
```

Comment:
> Oh we shouldn't do that @vadmeste

In all cases, we need to apply a transformation of that []byte because it is already inside a json response. For example, quoting ".